### PR TITLE
Added Singly Linked List implementation

### PR DESCRIPTION
**Pull request by Gagan-Shenoy**

### CODE REVIEW

**Comment by Gagan-Shenoy** [comment]
Please review my code @anku580 : )

File: linkedList/SinglyLinkedList.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
+package linkedlist;
+
+
+
```
Comment (anku580) [inline linkedList/SinglyLinkedList.java]: Well, this is a good learning. It is always best approach to not add any extra space. so, can you please remove the extra spaces?

**Review by anku580 — CHANGES_REQUESTED** [review]
I've added a comment, please address it.

**Comment by Gagan-Shenoy** [comment]
ok  @anku580 ill change it ,thanks for the feedback

**Comment by Gagan-Shenoy** [comment]
are the changes applied?

**Comment by Gagan-Shenoy** [comment]
@anku580 can i also add the search method for bst if you mind

**Review by anku580 — APPROVED** [review]
Thank you for contributing 👏

**Comment by anku580** [comment]
@Gagan-Shenoy Sure you can raise another PR for that.

**Pull request merged**